inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/sort/util/LoadNodeUtils.java

```diff
@@ -600,7 +601,7 @@ public static MySqlLoadNode createLoadNode(MySQLSink mysqlSink, List<FieldInfo>
                 null,
                 null,
                 properties,
-                mysqlSink.getJdbcUrl(),
+                mysqlSink.getJdbcUrl() + InlongConstants.SLASH + mysqlSink.getDatabaseName(),
```

Comment:
If jdbcUrl need add other params, this string append can cause exception

Reply:
I made some changes, you can look at it again.

Reply:
ok